classification/settings.py

```diff
@@ -21,21 +21,25 @@
 # ----------------------------------------------------------------------
 
 
-SIGNAL_TYPES = ["white_noise"]
-RESULTS_DIR = "results"
-DATA_DIR = "data" 
-MODEL_PARAMS_DIR = 'model_params'
+# Parameters to generate the artificial sensor data
+OUTFILE_NAME = "white_noise"
```

Comment:
agreed. renamed to "sensor_data_exp_settings.py" since i'll be using more than white noise for the experimments in the future.